docker-installer/install_cni_plugins.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+set -e
+
+if [ $# -ne 2 ]; then
+  echo "USAGE: $0 source-dir dest-dir"
+  exit 1
+fi
+
+SRC=$1
+DST=$2
+
+install_cni_plugin() {
+	NAME=$1
+	MD5=$( md5sum $SRC/$NAME | awk '{ print $1 }' )
+	if [ -e $DST/$NAME ]; then
+		if [ ! -e $DST/$NAME.by_cni_installer_image ]; then
+			# The file already exists but there's no marker that
+			# it was installed by this installer -> keep untouched
+			echo "* '$NAME' ignored (exists but not installed by me)"
+			return
+		fi
+		OTHER_MD5=$( md5sum $DST/$NAME | awk '{ print $1 }' )
+		INSTALLED_MD5=$( cat $DST/$NAME.by_cni_installer_image )
+
+		if [ "$OTHER_MD5" != "$INSTALLED_MD5" ]; then
+			# The file was previously installed by this installer
+			# but later changed -> keep untouched
+			echo "* '$NAME' ignored (previously installed by me but changed by someone else)"
```

Comment:
thought: how about a FORCE environment variable that continues here?

Reply:
I'm trying to think as a user who uses `FORCE=1`. The user might expect all plugins to be overridden.

Reply:
My first thought would be a user that has really old CNI binaries from an unknown source. They might be surprised that they aren't being upgraded.

Reply:
done in https://github.com/containernetworking/plugins/pull/844#discussion_r1124322021